src/main/resources/com/mathworks/ci/RunMatlabBuildStep/help-buildOptions.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  Specify a space-separated list of MATLAB build options. The plugin supports
+  the same build options that you can pass to the <b>buildtool</b> command when
+  running a MATLAB build.<br />
```

Comment:
That sounds reasonable. But the help text here seems to align with the one which we have for [matlab-actions](https://github.com/matlab-actions/run-build?tab=readme-ov-file#run-matlab-build) as discussed with Houman. Tagging @mw-hrastega for review!

Reply:
Good discussion! Sam's suggestions work well here because this is UI help text and reads better when summarized. Among the two alternatives in Sam's first paragraph, I would go with "... the build options that you can pass to the <b>buildtool</b> command "